packages/graphql-cli-validate-magento-pwa-queries/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "graphql-cli-validate-magento-pwa-queries",
+  "version": "1.0.0",
```

Comment:
Should we match this version to the others?

Reply:
I don't rightly know ... I don't think so?  I'm not sure why they're all at the same version in the first place to be honest though.

Reply:
Packages can and should be independently versioned. We're only versioning them in lockstep right now because of the historical locking with lerna.